source/mainnet/docs/guides/export-transaction-logs.rst

```diff
@@ -10,19 +10,13 @@ It is possible to export tansaction logs with :ref:`CCD Scan<ccd-scan>` from the
 
 .. dropdown:: |cryptox|
 
-    #. Tap the account.
-
-    #. On the Account screen, tap |cryptoX-acct-settings| to open the Account settings menu.
+    #. Go to the Your Accounts screen (tap the account identifier on the main screen).
```

Comment:
Then "Open Account Settings by tapping the gear icon in the upper right of the screen."